src/compile.ts

```diff
@@ -20,6 +20,7 @@ export async function compile(
   directory: string,
   ignore: string[] = [],
   solcModule: string = 'solc',
+  solcSettings: object = {optimizer: {enabled: true, runs: 200}},
```

Comment:
Running the optimizer by default will only add to the running time for no benefit since we are only interested in the AST.

```suggestion
  solcSettings: object = {},
```